door-runtime/src/jsMain/kotlin/com/ustadmobile/door/SimpleDoorQuery.kt

```diff
@@ -5,15 +5,56 @@ import com.ustadmobile.door.jdbc.PreparedStatement
 
 actual class SimpleDoorQuery actual constructor(private val sql: String, override val values: Array<out Any?>?) : DoorQuery {
 
+    val SQL_COMPONENT_TYPE_MAP = mapOf(Long::class to "BIGINT",
```

Comment:
This seems to have been copy/pasted in. Why not put simpledoorquery in commonJvmJs?